### ISSUE TITLE
ReCaptcha with chromedriver on Mac can't be beaten

### ISSUE THREAD

**Issue by earlvanze** [ChrisMuir/Zillow#13]
I've spent the last several minutes solving reCaptcha problems. It won't go away. Somehow Zillow or Google know Chrome is being controlled by automated test software (chromedriver).

Finally it gave up and told me "Try again later
Your computer or network may be sending automated queries. To protect our users, we can't process your request right now. For more details visit our help page"

**Comment by ChrisMuir** [comment]
Hi @earlvanze , yeah I've seen this happen in the past, see issue [#9](https://github.com/ChrisMuir/Zillow/issues/9). The last time I tested this, it was fairly rare for the CAPTCHA to continuously reload like you're describing. Typically it goes away after it's beaten the first time. Is that the case for you, or are you seeing it continuously reload EVERY time it comes up?

**Comment by earlvanze** [comment]
For me, I've never been able to beat it so I just gave up after trying multiple times. It reloads EVERY time after I click Verify over and over. Google has gotten smarter.

[Earl Co - Contact Using Hop](http://GetHop.com/?_hmid=1525541072)

On May 5, 2018 at 15:45 GMT, Chris Muir <notifications@github.com> wrote:

Hi [@earlvanze](https://github.com/earlvanze) , yeah I've seen this happen in the past, see issue [#9](https://github.com/ChrisMuir/Zillow/issues/9). The last time I tested this, it was fairly rare for the CAPTCHA to continuously reload like you're describing. Typically it goes away after it's beaten the first time. Is that the case for you, or are you seeing it continuously reload EVERY time it comes up?

**Comment by ChrisMuir** [comment]
Hmm, yeah sorry to hear this. As I said in issue [#9](https://github.com/ChrisMuir/Zillow/issues/9):
>The short answer is that once the CAPTCHA appears, it's out of my hands. I have no interest in developing the current CAPTCHA code beyond what it currently is, which is simply to pause code execution indefinitely until the CAPTCHA been manually handled.

Thank you for reporting this, but I am closing this issue.

**Comment by earlvanze** [comment]
Yeah I figured, thanks anyway. It's good code, I watched it work for a few seconds.

[Earl Co - Contact Using Hop](http://GetHop.com/?_hmid=1525542703)

On May 5, 2018 at 17:35 GMT, Chris Muir <notifications@github.com> wrote:

Hmm, yeah sorry to hear this. As I said in issue [#9](https://github.com/ChrisMuir/Zillow/issues/9):

The short answer is that once the CAPTCHA appears, it's out of my hands. I have no interest in developing the current CAPTCHA code beyond what it currently is, which is simply to pause code execution indefinitely until the CAPTCHA been manually handled.

Thank you for reporting this, but I am closing this issue.
